ETHOS.md

```diff
@@ -0,0 +1,81 @@
+
+> "Some people think they are in community, but they are only in proximity. True community requires commitment and openness. It is a willingness to extend yourself to encounter and know the other." - David Spangler
+
+We all come from different backgrounds; arbritrary differences such as ethnicity, gender, sexuality, religion, ability/lack-there-of or status should never have any influence on deciding whether someone should be treated with respect.
+
+We can all have a moment of idiocy where we say something, or act in a way, which is less than what we would expect from others to do unto ourselves. That's why we have a system which makes sure (to the best of our ability) that *everyone* is held accountable by someone else to a set of values that are agreed upon. It's more of a tool to be used to remind someone when their behaviour is not in line with the values that **you** and **I** have agreed upon by deciding to participate in this community.
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## Code of Ethics
+
+1. We will not discriminate against any people(s); this includes discrimination by virtue of race, gender(s), sexuality, age, economic standing, education, or disability (physical or otherwise)
+2. We will not knowingly try to harm any people(s) - physical or otherwise
+3. We must endeavour to be open, transparent and honest
+4. We believe in freedom of information
+5. We believe in freedom of speech where it is not in conflict with any other ethos or value
+6. We must be accountable for our actions
+7. We will be impartial and independent of any other allegiance(s)
+8. We believe that disagreement is a natural occurance; constructive criticism, and intellectual debate are welcomed; however, beligerant speech and aggression are neither
+
+<!-- 
+### Action
+
+Please see the document relating to actions that can be taken against breaches in the CODE_OF_CONDUCT.md
+
+If you have any concerns, they can be raised privately with a moderator. If you are concerned that unjust action has been taken against you by a moderator or a member of a team, then you can contact the delegated mediator [NAME] who is authorised to handle these disagreements.
+
+Confidentiality can be provided when requested, where reasonable, and within the means available.
+
+We measure our success by the amount of times we reach this stage (the less the better), but we are always trying to do better; do not be afraid to reach out.
+ -->
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+### Scope
+
+The code of ethics/ethos applies to all activities involving Nim-Works. It forms the basis for our code of conduct and decisions where we are otherwise not explicit.
```

Comment:
```suggestion
The code of ethics applies to all activities involving Nim-Works. It forms the basis for our code of conduct and decisions where we are otherwise not explicit.
```

Relatedly, but not a blocker I'd prefer renaming this file to `ETHICS.md`, as this reads as principles (abstract), rather than as 'customers and character' (imagery).